example/.wordpress-network

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env hocker
+# vim:set ft=sh:
+
+hocker_network_create wordpress \
+	--driver overlay \
+	--attachable
```

Comment:
Ok, you convinced me -- switched to `bridge`, and even gave our interface a pretty name. :smile: